mask_point_cloud.py

```diff
@@ -0,0 +1,129 @@
+"""Import ZDF point cloud, apply a binary mask, and visualize it."""
+
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+import pptk
+import zivid
+
+
+def _display_rgb(rgb):
+    """Display RGB image.
+
+    Args:
+        rgb: RGB image
+
+    """
+    plt.figure()
+    plt.imshow(rgb)
+    plt.title("RGB image")
+    plt.show(block=False)
+
+
+def _display_depthmap(xyz):
+    """Create and display depthmap.
+
+    Args:
+        xyz: X, Y and Z images (point cloud co-ordinates)
+
+    """
+    plt.figure()
+    plt.imshow(
+        xyz[:, :, 2],
+        vmin=np.nanmin(xyz[:, :, 2]),
+        vmax=np.nanmax(xyz[:, :, 2]),
+        cmap="jet",
+    )
+    plt.colorbar()
+    plt.title("Depth map")
+    plt.show(block=False)
+
+
+def _get_mid_point(xyz):
+    """Calculate mid point from average of the 100 centermost points.
+
+    Args:
+        xyz: X, Y and Z images (point cloud co-ordinates)
+
+    Returns:
+        mid_point: Calculated mid point
+
+    """
+    xyz_center_cube = xyz[
+        int(xyz.shape[0] / 2 - 5) : int(xyz.shape[0] / 2 + 5),
```

Comment:
Done. It is a kind of offset. You get 100 central pixels by taking all pixels from position 5 left and 5 up to 5 right and 5 down.

Reply:
These should probably have a name, since these numbers are now magic numbers. Giving them a label helps the reader of the code where these numbers came from, and also reduces code duplication